cppwinrt/component_writers.h

```diff
@@ -1208,4 +1208,4 @@ namespace winrt::@::implementation
                 slot);
         }
     }
-}
```

Comment:
Please leave a new line at the end of each file. Some compilers don't like it otherwise.